sbpy/activity/gas/productionrate.py

```diff
@@ -495,126 +619,212 @@ def from_Drahus(self, integrated_flux, mol_data, ephemobj, vgas=1 * u.km/u.s,
 
         return q
 
-    def from_Haser(self, coma, mol_data, aper=25 * u.m):
+
+class NonLTE():
+    """
+    Class method for non LTE production rate models
+    Not Yet implemented
+
+    """
+
+    def from_pyradex(self, integrated_flux, mol_data, line_width=1.0 * u.km / u.s,
+                     escapeProbGeom='lvg', iter=100,
+                     collider_density={'H2': 900*8.934}):
```

Comment:
Here the "standard" scaling relation for collisional rates is based on the reduced mass of the collision system. For deriving H2O-H2O collision rates from H2-H2O coefficients we would apply the square root of the ratio of reduced masses `(mu_H2O/mu_H2)**0.5 = ((18**2/18*2)/((18*2)/(18+2)))**0.5 = 2.2`. This factor reduces the H2-H2O collision rates (dividing rather than multiplying) because of the slower collisions. See the discussion in [this paper](https://arxiv.org/abs/1406.5750) for the case of He-H2 collision rates. The authors conclude that this scaling is not very reliable but probably is our best option.

Reply:
Got it, thanks!